stream-chat-android-offline/build.gradle

```diff
@@ -19,21 +19,24 @@ ext {
 apply from: "${rootProject.projectDir}/scripts/publish-mavencentral.gradle"
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion Configuration.compileSdkVersion
     buildToolsVersion "29.0.3"
     defaultConfig {
-        minSdkVersion 21
-        targetSdkVersion 28
-        versionCode 1
+        minSdkVersion Configuration.minSdkVersion
```

Comment:
What do you think, is it a good idea to apply `buildToolsVersion`, `compileSdkVersion`, `compileOptions`, `minSdkVersion`, `targetSdkVersion`, (optionally kotlin source sets) to all subprojects in the root level gradle build script?

Reply:
Should be possible - there's a whole bunch of things that we could extract into common configuration. Don't want to touch too many things for now, just wanted to apply the quick fix of using the configured values instead of individual hardcoded ones.